src/aat/resources/features/F-042/F-042_Test_Data_Base.td.json

```diff
@@ -0,0 +1,21 @@
+{
+  "_guid_": "F-042_Test_Data_Base",
+  "productName": "CCD Data Store",
+  "operationName": "Start the event creation process for a new case for a Case Worker",
+  "method": "GET",
+  "uri": "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/{cid}/event-triggers/{etid}/token",
+  "user": {
+    "username": "befta.caseworker.1@gmail.com",
```

Comment:
does setting the user like this still work? I thought we updated the DSL implementation to take the user credentials like this: "users": { "invokingUser": { "username": "abc", "password": "abc" } }

Reply:
The user setting was working(surprisingly). I have updated it to include an "invokingUser" json node.